src/core/templates/navigation.hbs

```diff
@@ -1,7 +1,12 @@
 {{! make the _globals object in course.json available to this template}}
 {{import_globals}}
 {{! All links in the nav bar should have an attribute of data-event, navigationView uses this to fire a global event}}
-<div class="navigation-inner clearfix" role="navigation" aria-label="{{_globals._accessibility._ariaLabels.navigation}}">
+<div class="navigation-inner clearfix" role="navigation">
+	{{#if _accessibility._isEnabled}}
+		<button id="accessibility-toggle" class="base icon icon-accessibility a11y-ignore-focus"></button>
+		<span id="accessibility-instructions" class="aria-label" role="region" tabindex="0"></span>
+	{{/if}}
+	{{{a11y_aria_label _globals._accessibility._ariaLabels.navigation}}}
```

Comment:
Minor, but think the indentation is a bit off here.

Reply:
Tabs crept in - have converted to spaces